mdakit_membcurv/core.py

```diff
@@ -5,7 +5,7 @@
 Handles the primary functions
 """
 
-from mods import *
+from .lib.mods import *
```

Comment:
I would avoid blanket `from x import *` imports but be explicit about which functions you actually need here. It makes the code a lot easier to debug and to read. Furthermore, if you have two blanket imports, _nobody_ (including future you) will likely have an idea where the functions really came from.

(I am not even a fan of `from x import y` and prefer to use `import y` and then use `y.x` to make clear where it comes from, or use a variation with submodule imports and aliasing like `from matplotlib import pyplot as plt`.)

Reply:
Thanks for this. I'll leave it as absolute import in the current test as `from mods import y` bit I will keep in mind the change as you suggested in the refactoring process.